apps/admin-ui/src/identity-providers/add/OIDCAuthentication.tsx

```diff
@@ -76,6 +82,50 @@ export const OIDCAuthentication = ({ create = true }: { create?: boolean }) => {
         secretRequired={clientAuthMethod !== "private_key_jwt"}
         create={create}
       />
+      <FormGroup
+        label={t("clientAssertionSigningAlg")}
+        labelIcon={
+          <HelpItem
+            helpText="identity-providers-help:clientAssertionSigningAlg"
+            fieldLabelId="identity-providers:clientAssertionSigningAlg"
+          />
+        }
+        fieldId="clientAssertionSigningAlg"
+      >
+        <Controller
+          name="config.clientAssertionSigningAlg"
+          defaultValue=""
+          control={control}
+          render={({ field }) => (
+            <Select
+              maxHeight={200}
+              toggleId="clientAssertionSigningAlg"
+              onToggle={() => setOpenClientAuthSigAlg(!openClientAuthSigAlg)}
+              onSelect={(_, value) => {
+                field.onChange(value.toString());
+                setOpenClientAuthSigAlg(false);
+              }}
+              selections={field.value || t("algorithmNotSpecified")}
+              variant={SelectVariant.single}
+              aria-label={t("signatureAlgorithm")}
+              isOpen={openClientAuthSigAlg}
+            >
+              <SelectOption selected={field.value === ""} key="" value="">
+                {t("algorithmNotSpecified")}
+              </SelectOption>
+              <>
+                {providers.map((option) => (
+                  <SelectOption
+                    selected={option === field.value}
+                    key={option}
+                    value={option}
+                  ></SelectOption>
```

Comment:
This element can be self-closing.
```suggestion
                  />
```

Reply:
I fixed it.